src/server/pps/server/api_server_test.go

```diff
@@ -459,6 +459,61 @@ func TestCreatePipelineV2(t *testing.T) {
 	require.False(t, req.Autoscaling, "spec must override default")
 }
 
+func TestRerunPipeline(t *testing.T) {
+	ctx := pctx.TestContext(t)
+	env := realenv.NewRealEnv(ctx, t, dockertestenv.NewTestDBConfig(t))
+
+	repo := "input"
+	pipeline := "pipeline"
+	require.NoError(t, env.PachClient.CreateRepo(pfs.DefaultProjectName, repo))
+	var pipelineTemplate = `{
+		"pipeline": {
+			"project": {
+				"name": "{{.ProjectName | js}}"
+			},
+			"name": "{{.PipelineName | js}}"
+		},
+		"transform": {
+			"cmd": ["cp", "r", "/pfs/in", "/pfs/out"]
```

Comment:
I think this command should run something like "date" so the test can assure that it reran.